lmms_eval/tasks/video-tt/gpt_utils.py

```
@@ -0,0 +1,269 @@
+    """
+    # try:
+    #     question = doc["question"]
+    #     answer = doc["answer"]
+    #     pred = result[0]
+
+    #     # Assume get_eval returns a review and the model name, and parse_score parses this review
+    #     review, model_name = get_eval(question, answer, pred, 1)
+    #     scores = parse_score(review)
+    # except Exception as e:
+    # eval_logger.error(f"Error for Question ID: {doc.get('question_id', 'Unknown')}: {e}")
+    question = doc["question"]
```

Ruff (0.11.9)

Local variable `question` is assigned to but never used

Remove assignment to unused variable `question`

(F841)